testing/btest/scripts/base/frameworks/logging/sqlite/simultaneous-writes.zeek

```diff
@@ -2,6 +2,8 @@
 #
 # @TEST-REQUIRES: which sqlite3
 # @TEST-REQUIRES: has-writer Zeek::SQLiteWriter
+# Don't run this test if we build with '--sanitizers=thread'
```

Comment:
```suggestion
# Don't run this test if we build with '--sanitizers=thread' because we disable the shared cache in that case due to a SQLite bug.
```